### PR TITLE
add browser based notifications

### PR DESCRIPTION
**Pull request by pranay414**
Added browser based notifications on successful file upload

Fixes issue https://github.com/NITDgpOS/PiZilla/issues/38

### CODE REVIEW

File: app/components/DropZone.jsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import DropzoneComponent from 'react-dropzone-component';
 import FilesActions from './../actions/FilesActions';
 import PropTypes from 'prop-types';
 
+Notification.requestPermission();
```
Comment (naveenkumarsangi) [inline app/components/DropZone.jsx]: The request for permission should be done in the constructor. Also, if the user has already rejected the permission, stop bugging him/her again.

**Review by naveenkumarsangi — CHANGES_REQUESTED** [review]

**Comment by nkprince007** [comment]
@gitmate-bot rebase

**Comment by naveenkumarsangi** [comment]
@gitmate-bot rebase

**Comment by naveenkumarsangi** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by naveenkumarsangi** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by naveenkumarsangi** [comment]
unack https://github.com/NITDgpOS/PiZilla/pull/69/commits/1d40d2653a1220fcd3aa131e588b5e66352b64e2

Please address the requested changes. @pranay414

**Comment by pranay414** [comment]
@nkprince007 I am not clear on what you mean by unack 1d40d26, can you explain it?

**Comment by nkprince007** [comment]
Sorry @pranay414, you do not have the necessary permission levels to perform the action.

File: app/components/DropZone.jsx (hunk 1 above)
Comment (pranay414) [reply app/components/DropZone.jsx]: is there any advantage in doing so?

**Comment by naveenkumarsangi** [comment]
Uhm, that's a part of GitMate.io application, that allows me to accept and reject commits with the commands `ack` and `unack`. You don't need to worry about that. Once I'm happy, I'll be `ack` or acknowledging your commits.

And the bot is using my-account to post comments. So, it may seem like that I'm doing so, but it is infact a bot.

File: app/components/DropZone.jsx (hunk 1 above)
Comment (naveenkumarsangi) [reply app/components/DropZone.jsx]: We shouldn't write arbitrary code outside the react components. It is error-prone. It should be a part of the component life-cycle.

**Comment by naveenkumarsangi** [comment]
@gitmate-bot rebase.

**Comment by naveenkumarsangi** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by naveenkumarsangi** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by naveenkumarsangi** [comment]
Never merge `master` into your branches. We do not want merge commits. They mess up history. Refer https://coala.io/rebase for more info.

I've rebased your PR with GitMate for now. But make sure you stick to that philosophy. Because most of the git based projects prefer linear commit history.

Thanks for the PR, By the way 🎉

**Review by naveenkumarsangi — APPROVED** [review]

**Comment by naveenkumarsangi** [comment]
ack https://github.com/NITDgpOS/PiZilla/pull/69/commits/c54f5756a9b90aab01dfdf9f417dcfebeb9a6a4d

**Comment by nkprince007** [comment]
@gitmate-bot fastforward

**Comment by nkprince007** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by nkprince007** [comment]
Automated fastforward failed! Please fastforward your pull request manually via the command line.

Reason:
```
Command: git push origin HEAD:master

Exit Code: 1

Cause:

To <hidden_oauth_token>
 ! [remote rejected] HEAD -> master (permission denied)
error: failed to push some refs to '<hidden_oauth_token>'


```

**Pull request merged**